src/device-registry/config/global/db-projections.js

```
@@ -191,16 +181,32 @@ class ProjectionFactory {
           isOnline: 1,
           activities: "$activities",
           latest_deployment_activity: {
-            $arrayElemAt: ["$latest_deployment_activity", 0],
+            $cond: {
+              if: { $isArray: "$latest_deployment_activity" },
+              then: { $arrayElemAt: ["$latest_deployment_activity", 0] },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)